CHANGELOG.md

```diff
@@ -12,34 +12,48 @@ All notable changes to this project will be documented in this file.
 "### Fixed" for any bug fixes.
 "### Security" in case of vulnerabilities.
 -->
+
+## [5.0.0] - 2021-06-xx
+
+### Changed
+- [BREAKING] Upgraded the component to work with Angular 8
```

Comment:
is dit echt breaking? als ik de wijzigingen zie lijkt het me dat je de nieuwe versie ook nog met de oude core branding moet kunnen gebruiken?

Reply:
Als je remote data opvraagt, verschijnt er normaal gezien een spinner in de autocomplete die niet meer werkt.